docs/settings/rules/ears-format.md

```
@@ -0,0 +1,49 @@
+# EARS Format Guidelines
+
+## Overview
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)